function getErrorComplete(action, error){
    console.log(error);
}

module.exports = getError